subxt/src/backend/legacy/mod.rs

```diff
@@ -431,6 +506,11 @@ impl<T: Config> Stream for StorageFetchDescendantKeysStream<T> {
                         return Poll::Ready(Some(Ok(keys)));
                     }
                     Err(e) => {
+                        if e.is_disconnected_will_reconnect() {
```

Comment:
Perhaps worth a comment, because this is doing something a bit weird: if a future returns an `Err(DisconnectedWillReconnect)`, we expect to _keep polling it_ and it will internally retry itself :)

Reply:
true, changed it to continue the loop instead